pippy_app.py

```diff
@@ -539,8 +539,35 @@ def _load_example_cb(self, widget):
         self._dialog.run()
         path = self._dialog.get_path()
         if path:
+            if path.endswith('physics') and not self._ensure_box2d_support():
+                return
             self._select_func_cb(path)
 
+    def _ensure_box2d_support(self):
+        try:
+            import Box2D
+        except ImportError:
+            dialog = Gtk.MessageDialog(
+                None, Gtk.DialogFlags.MODAL,
+                Gtk.MessageType.WARNING,
+                Gtk.ButtonsType.NONE,
+                _('Box2D is not installed on your system'))
+            dialog.format_secondary_text(
+                _('Box2D is required to run this example. This example will fail to run. Do you still wish to continue?'))
+            dialog.add_button("Force Load", Gtk.ResponseType.OK)
+            dialog.add_button("Cancel", Gtk.ResponseType.CANCEL)
```

Comment:
The only appropriate response to Box2D not being installed is an okay as a "Force Load" doesn't change anything.

Also you can remove the "Do you still wish to continue?".